### DISCUSSION TITLE
Removing an app from PocketHost?

### DISCUSSION THREAD

**Discussion by oksure** [pockethost/pockethost#133] [Q&A]
Is there a way for me to delete an app from my dashboard? I created one to test out, and now I am thinking of creating an app for real use cases. It's not a deal breaker, but it'd be nice not to live with clutters I had created for testing.

Thanks, and please keep up the great work!

**Comment by benallfree** [comment]
Hello, that is a frequently requested feature. We have it planned, you can track progress in #35. 

Thanks for the suggestion!
